bsp/nuclei/hbird_eval/applications/main.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2006-2018, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ * 2019-07-23     tyustli      first version
+ */
+
+#include <rtthread.h>
+#include <rtdevice.h>
+#include <nuclei_sdk_hal.h>
+
+#define THREAD_PRIORITY 19
```

Comment:
better to keep the demo as simple as possible.

Reply:
I can keep a simple demo.